# People and Contacts

The Cancer Genome Atlas (TCGA) is coordinated by a project team comprised of individuals from both the National Cancer Institute and the National Human Genome Research Institute and advised by an External Scientific Committee whose membership includes patient advocates, senior scientists and clinicians with relevant expertise in cancer, genomics and ethics.  
    
## About Contacting TCGA
- TCGA cannot share samples or analytes from the TCGA project with any external entities.
- The program office does not provide help with bioinformatics or guidance on independent research projects that use TCGA data.
- If you have a question about data in the Genomic Data Commons, please contact support@nci-gdc.datacommons.io
    
Select a link below for more information:
    
- TCGA Project Team
- TCGA External Scientific Committee
- TCGA Program Office

## How do I cite NCBI services and databases?
To acknowledge NCBI services in your writing, you can refer to (1) publications that NCBI staff publishes in peer-reviewed journals or (2) to the NCBI web site (internet) and individual services/records within the site.
 
1. Publications describing NCBI services in peer-reviewed journals:
As a general reference, use the Database Resources of the National Center for Biotechnology Information article published in Nucleic Acids Research (NAR). Use the Citation link on the right side of the PMC view of this article to obtain the citation in the desired format.
Check through this PubMed listing to locate citations/papers on individual NCBI services of your interest.
For the BLAST® services, use these BLAST references.

2. Citing the NCBI internet site and individual web pages and records:
To cite the entire NCBI site, use this format:
National Center for Biotechnology Information (NCBI)[Internet]. Bethesda (MD): National Library of Medicine (US), National Center for Biotechnology Information; [1988] – [cited 2017 Apr 06]. Available from: https://www.ncbi.nlm.nih.gov/
(The date within the square brackets represents your date of citing the site.)
To cite an individual service or database, format your citation according to this example:
Gene [Internet]. Bethesda (MD): National Library of Medicine (US), National Center for Biotechnology Information; 2004 – [cited YYYY Mmm DD]. Available from: https://www.ncbi.nlm.nih.gov/gene/
To cite an individual record, format your citation according to this example:
Nucleotide [Internet]. Bethesda (MD): National Library of Medicine (US), National Center for Biotechnology Information; [1988] – . Accession No. NM_001349333.1, Homo sapiens FYN binding protein (FYB), transcript variant 4, mRNA; [cited YYYY Mmm DD]. Available from: https://www.ncbi.nlm.nih.gov/nuccore/NM_001349333.1

Pay attention to the correct accession format when citing individual records.


For more details also see:
The attached file that contains:
examples for citing some of the popular NCBI services
additional guidance on citing GenBank and RefSeq records
Chapter 24 Databases/Retrieval Systems/Datasets on the Internet in the Citing Medicine guide.

## What are accession numbers?

In addition to unique record identifiers, NCBI staff (or collaborators) often assign accession numbers to database records or individual units of data. You will encounter accession numbers mostly in databases that serve as primary repositories of sequence and other molecular data.
 
1. Several NCBI databases use the following format for accession numbers: [alphabetical prefix][series of digits]

For example, PRJNA318322 is an accession number for a record in the BioProject database. Other databases with similar record accessioning are: BioSample, Sequence Read Archive (SRA), GEO DataSets, dbSNP, dbVar, and Probe. 
 
2. Accession formats for sequence records include a version number: [alphabetical prefix][series of digits][.][version number]

Version numbers are assigned for those types of sequence data that we expect to be updated over time. For example, the version suffix "3" in accession NM_000680.3 indicates that the sequence in the record has been updated twice. The version increments only if the sequence itself is updated, and does not change for updates to any other fields, such as publication lists, author names, and feature annotation on the sequence.

The formats of sequence accession numbers are of distinct types, depending on the NCBI database. The alphabetical prefixes contain various embedded information. Select a database link below for details on its accession numbers:

+ Nucleotide: GenBank (INSDC)
+ Nucleotide: Reference Sequences (RefSeq)
+ Protein
+ Assembly